conjure/steps/01_get-kubectl.sh

```diff
@@ -20,7 +20,7 @@ if [ $(unitStatus kubernetes 0) != "active" ]; then
 fi
 
 # TODO: Convert to an actionable item
-juju scp kubernetes/0:kubectl_package.tar $KUBECTL_PATH/.
-cd $KUBECTL_PATH && tar zxf kubectl_package.tar
+juju scp kubernetes/0:kubectl_package.tar.gz $KUBECTL_PATH/.
```

Comment:
I'm thinking I'll add something in our hook library within conjure-up to grab the leader